### ISSUE TITLE
Error in refresh menu item

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode-pull-request-github#78]
Related to #77

1. Open a new window without folder
2. In SCM, click the refresh menu item

```
Error: command 'pr.refreshList' not found
    at CommandService._tryExecuteCommand (file:///Users/joao/Work/vscode/out/vs/workbench/services/commands/common/commandService.js:66:56)
    at file:///Users/joao/Work/vscode/out/vs/workbench/services/commands/common/commandService.js:60:65
    at Object.notifySuccess [as _notify] (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1185:59)
    at Object.enter (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:867:30)
    at Promise_ctor._run (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1083:29)
    at Promise_ctor._completed (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1051:18)
    at argDone (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1828:37)
    at Promise.then.errors.(anonymous function) (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1853:78)
    at Object.notifySuccess [as _notify] (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1185:59)
    at Object.enter (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:867:30)
    at Promise_ctor._run (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1083:29)
    at Promise_ctor._completed (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1051:18)
    at argDone (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1828:37)
    at Promise.then.errors.(anonymous function) (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1853:78)
    at Object.notifySuccess [as _notify] (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1185:59)
    at Object.enter (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:867:30)
    at _Base.Class.derive._creator._run (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1083:29)
    at _Base.Class.derive._creator._completed (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1051:18)
    at Object.notifySuccess [as _notify] (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1185:59)
    at Object.enter (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:867:30)
    at Promise_ctor._run (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1083:29)
    at Promise_ctor._completed (file:///Users/joao/Work/vscode/out/vs/base/common/winjs.base.js:1051:18)
    at LazyPromise.resolveOk (file:///Users/joao/Work/vscode/out/vs/workbench/services/extensions/node/lazyPromise.js:43:22)
    at RPCProtocol._receiveReply (file:///Users/joao/Work/vscode/out/vs/workbench/services/extensions/node/rpcProtocol.js:181:26)
    at RPCProtocol._receiveOneMessage (file:///Users/joao/Work/vscode/out/vs/workbench/services/extensions/node/rpcProtocol.js:145:26)
    at file:///Users/joao/Work/vscode/out/vs/workbench/services/extensions/node/rpcProtocol.js:76:92
    at file:///Users/joao/Work/vscode/out/vs/workbench/services/extensions/node/rpcProtocol.js:249:21
    at Emitter.fire (file:///Users/joao/Work/vscode/out/vs/base/common/event.js:148:38)
    at acceptChunk (file:///Users/joao/Work/vscode/out/vs/base/parts/ipc/node/ipc.net.js:90:46)
    at Socket.<anonymous> (file:///Users/joao/Work/vscode/out/vs/base/parts/ipc/node/ipc.net.js:113:17)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:560:20)
```

**Comment by RMacfarlane** [comment]
Should be fixed now that #77 is fixed, refresh command won't show up in this case, only when the tree view that registers it is shown

**Comment by anisjonischkeit** [comment]
I am still experiencing this issue.
